platform/debian/kanidm-unixd/postinst

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+# postinst script for kanidm-unixd
+#
+# see: dh_installdeb(1)
+
+set -e
+
+
+case "$1" in
+    configure)
+        if [ ! -f /etc/kanidm/config ]; then
```

Comment:
won't this run multiple times for -ssh and -unixd?

Reply:
sure will, there's one post-inst script per .deb - hence the check for the config file

Reply:
Are these all part of "one deb"? Or is -unixd and -ssh separet debs?